docker/simd/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG VERSION=v0.42.3
```

Comment:
Makefile should pass this arg from the go.mod and the tests should check the sdk version in go mod and build if the container doesn't exist.